packages/scandipwa/src/component/CategoryConfigurableAttributes/CategoryConfigurableAttributes.component.js

```diff
@@ -88,6 +89,30 @@ export class CategoryConfigurableAttributes extends ProductConfigurableAttribute
         );
     }
 
+    renderConfigurableAttributeValue(attribute) {
+        const {
+            getIsConfigurableAttributeAvailable,
+            handleOptionClick,
+            getLink,
+            isSelected,
+            show_product_count
+        } = this.props;
+
+        const { attribute_value } = attribute;
+
+        return (
+            <ProductAttributeValue
+              key={ attribute_value }
+              attribute={ attribute }
+              isSelected={ isSelected(attribute) }
+              isAvailable={ getIsConfigurableAttributeAvailable(attribute) }
+              onClick={ handleOptionClick }
+              getLink={ getLink }
+              showProductCount={ show_product_count }
```

Comment:
Also, can you please make it `isShowProductCount` boolean all vars must start with `is`.

Reply:
changed to ```isProductCountVisible```